docs/blogs/IntroGatewayAPI/GettingStarted.md

```diff
@@ -0,0 +1,148 @@
+
+Let’s now use Gateway API to create a Gateway and Route. In this example, we will convert a simple HTTP OpenShift Route into
+roughly equivalent Gateway API objects:
+
+```yaml
+apiVersion: v1
+kind: Route
+metadata:
+  name: http
+  namespace: demo-app
+spec:
+  host: www.http.example.com
+  to:
+    kind: Service
+    name: foo-app
+  port:
+    targetPort: 8080
+```
+
+First, let’s create and configure a namespace called `demo-gateway` that will have our Gateway deployment and service.
+This namespace will also need the `anyuid` scc to allow the gateway deployment (Envoy) to function:
+```yaml
+$ oc create namespace demo-gateway
+$ oc adm policy add-scc-to-group anyuid system:serviceaccounts:demo-gateway
+```
+
+Create the Gateway with a Listener that will listen for requests to `*.example.com`. By default, Istio will [automatically
+provision](https://istio.io/latest/docs/tasks/traffic-management/ingress/gateway-api/#automated-deployment) a gateway
+deployment and service with the same name upon creation of this Gateway object:
+```yaml
+$ oc apply -f - <<EOF
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: Gateway
+metadata:
+  name: gateway
+  namespace: demo-gateway
+spec:
+  gatewayClassName: istio
+  listeners:
+  - name: demo
+    hostname: "*.example.com"
+    port: 80
+    protocol: HTTP
+    allowedRoutes:
+      namespaces:
+        from: All
+EOF
+```
+
+Next, let’s create and configure a namespace called `demo-app` that will have our HTTPRoute object and our example application:
+```shell
+$ oc create namespace demo-app
+```
+
+Create a demo application deployment and service using `oc new-app`, our `cakephp-ingress-demo` project, and the `foo`
+git branch. This will create a simple server, `foo-app`, that will allow us to test connectivity. Then use the `oc rollout`
+command to wait for it to be rolled out:
+```shell
+$ oc new-app -n demo-app --name foo-app https://github.com/openshiftdemos/cakephp-ingress-demo#foo
+$ oc rollout status deployment -w -n demo-app foo-app
+```
+
+Create the HTTPRoute that will direct requests for `http.example.com` to our new backend server, `foo-app`:
+```yaml
+$ oc apply -f - <<EOF
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: HTTPRoute
+metadata:
+  name: http
+  namespace: demo-app
+spec:
+parentRefs:
+  - name: gateway
+    namespace: demo-gateway
+  hostnames: ["http.example.com"]
+  rules:
+  - backendRefs:
+    - name: foo-app
+      port: 8080
+EOF
+```
+
+Wait for the gateway deployment (Envoy) to be ready. Then set the ingress host variable `INGRESS_HOST` to the external
+load balancer hostname or IP:
+```shell
+$ oc wait -n demo-gateway --for=condition=ready gateways.gateway.networking.k8s.io gateway
+$ export INGRESS_HOST=$(oc get gateways.gateway.networking.k8s.io gateway -n demo-gateway -ojsonpath='{.status.addresses[*].value}')
+```
+
+Now let’s curl the route we just created using a HTTP Head request to only get the headers. The `app` response header
```

Comment:
In my mind, "a HTTP" reads funny, but it makes sense outside the USA.

Reply:
I learned (from a logic teacher) that you should decide on a/an based on the first word of an acronym, rather than how the acronym is pronounced. But the web is saying the opposite - it should be based on pronunciation, so it would be "an HIV patient", but "a HIPAA document".  I always **say** "an HTTP", or "an SSL" , but chose to **write** it like my teacher taught me.  I guess I need to change with the times, so I'll change it.